### PR TITLE
blog: Add v1.16.3 and v1.16.2 release anouncement

### PR DESCRIPTION
**Pull request by daipom**
* https://github.com/fluent/fluentd/blob/v1.16/CHANGELOG.md#release-v1163---20231114
* https://github.com/fluent/fluentd/releases/tag/v1.16.3

### CODE REVIEW

File: content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# Fluentd v1.16.3 and v1.16.2 have been released
+
+Hi users!
+
+We have released v1.16.3 ([changelog](https://github.com/fluent/fluentd/blob/v1.16/CHANGELOG.md#release-v1163---20231114)) on 2023-11-14.
+We also describe the content of v1.16.2 ([changelog](https://github.com/fluent/fluentd/blob/v1.16/CHANGELOG.md#release-v1162---20230714)) on 2023-07-14 here.
+
+In these releases, `in_tails` issues that have been reported for years are fixed!
```
Comment (kenhys) [inline content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md:8]:
```suggestion
In these releases, `in_tail` issues that have been reported for years are fixed!
```

File: content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+See the following PRs for more information.
+
+* `follow_inodes true` case: [#4208](https://github.com/fluent/fluentd/pull/4208)
+* `follow_inodes false` case: [#4327](https://github.com/fluent/fluentd/pull/4327)
+
+### Others
+
+* `in_tail`: Fix possible log duplication of [follow_inodes option](https://docs.fluentd.org/input/tail#follow_inodes). [#4237](https://github.com/fluent/fluentd/pull/4237), [#4239](https://github.com/fluent/fluentd/pull/4239)
+* `in_forward`: Fix corrupted data possibly breaking other data. [#4178](https://github.com/fluent/fluentd/pull/4178)
+* [ignore_same_log_interval option](https://docs.fluentd.org/deployment/system-config#ignore_same_log_interval): Fix cache size growing unlimitedly. [#4229](https://github.com/fluent/fluentd/pull/4229)
```
Comment (kenhys) [inline content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md:43]:
```suggestion
* [ignore\_same\_log\_interval option](https://docs.fluentd.org/deployment/system-config#ignore_same_log_interval): Fix cache size growing unlimitedly. [#4229](https://github.com/fluent/fluentd/pull/4229)
```

It seems that it requires to be escaped here.

File: content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md (hunk 1 above)
Comment (daipom) [reply content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md]: Thanks!

File: content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md (hunk 2 above)
Comment (daipom) [reply content/blog/20231114_fluentd-v1.16.2-v1.16.3-have-been-released.md]: Thanks! Sorry for overlooking it. Fixed.

**Review by kenhys — APPROVED** [review]
LGTM.

**Comment by daipom** [comment]
Thanks for your review!

**Pull request merged**